Makefile

```diff
@@ -139,6 +140,16 @@ ifndef ignore-not-found
   ignore-not-found = false
 endif
 
+.PHONY: cert-mgr
+cert-mgr: ## Install cert-manager
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MGR_VERSION)/cert-manager.yaml
+	kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
+
+.PHONY: rukpak
+rukpak: ## Install rukpak
+	kubectl apply -f https://github.com/operator-framework/rukpak/releases/latest/download/rukpak.yaml
+	kubectl wait --for=condition=Available --namespace=rukpak-system deployment/core --timeout=60s
+
```

Comment:
Add a TODO to swap this out with whatever we come up with for the "distribution" or install script or whatever that ends up being?

Reply:
Updated